packages/ui/examples/ComplexModule.js

```diff
@@ -31,7 +31,7 @@ const synthetixDefinition = defineModule("Synthetix", (m) => {
 
   const { router } = m.useModule(uniswap);
 
-  m.call(synthetixCore, "setUniswap", [router]);
+  m.call(synthetixCore, "setUniswap(boolean,address)", [false, router]);
```

Comment:
This should be just `setUniswap`. If this doesn't work, it's a bug.

Reply:
You mean we should never have a user putting in the parameters in `setUniswap(boolean,address)`?